firmware/biome.json

```
@@ -1,26 +1,27 @@
+    ],
+    "ignoreUnknown": false
   },
   "formatter": {
     "enabled": true,
     "indentStyle": "space",
     "indentWidth": 2
   },
-  "organizeImports": {
-    "enabled": true
-  },
+  "assist": { "actions": { "source": { "organizeImports": "on" } } },
```

Biome (1.9.4)

[error] Found an unknown key `assist`.

Known keys:


(deserialize)